.github/workflows/terraform-plan-env.yml

```diff
@@ -10,9 +10,32 @@ on:
   
 
 jobs:
+  fetch-versions:
```

Comment:
Lines 13-34 look like they are just a copy of `workflows/fetch-versions.yml... This should be removed since you're calling that workflow and referring to its outputs below!